test/new-e2e/tests/agent-platform/install/install.go

```diff
@@ -50,6 +50,9 @@ func Unix(t *testing.T, client *common.TestClient, options ...installparams.Opti
 		apikey = params.APIKey
 	} else {
 		apikey = "aaaaaaaaaa"
+
+		// If the API key is not provided, disable the telemetry to avoid 403 errors
+		commandLine += fmt.Sprint("DD_INSTRUMENTATION_TELEMETRY_ENABLED=false")
```

Comment:
I'd add a space before&after to be sure it doesn't merge with another variables. Currently it works, but if we add a new variable or parameter it might start failing unexpectedly 
LGTM otherwise!